stubs/aiofiles/@tests/stubtest_allowlist.txt

```diff
@@ -93,3 +93,6 @@ aiofiles.tempfile.temptypes.AsyncTemporaryDirectory.cleanup
 # Metaclass differs:
 aiofiles.base.AiofilesContextManager
 aiofiles.tempfile.AiofilesContextManagerTempDir
+
+# Internal wrapper, not meant to be re-exported
+aiofiles.ospath.wrap
```

Comment:
https://github.com/python/typeshed/actions/runs/7121014427/job/19389429006?pr=11112#step:5:61

I'm surprised that stubtest thinks this needs to be a re-exported symbol. Usually it only does that with unused imports.

Reply:
Probably aiofiles is doing some really dynamic thing that screws up subtest's heuristics for figuring out if something was actually defined in this module, or originated from another module

Reply:
The heuristics are here: https://github.com/python/mypy/blob/1cdeecdcc11115a3d15610801fdf91b0ed6bbe0a/mypy/stubtest.py#L376